src/domain/collaboration/callout/comments/CommentsCallout.tsx

```diff
@@ -160,25 +149,16 @@ const CommentsCallout = forwardRef<HTMLDivElement, CommentsCalloutProps>(
               loading={loading || postingComment || deletingMessage}
               last={lastMessageOnly}
               maxHeight={COMMENTS_CONTAINER_HEIGHT}
-              onClickMore={() => setIsFullViewDialogOpen(true)}
+              onClickMore={() => {}}
             />
           </CalloutLayout>
         </PageContentBlock>
         <Dialog
-          open={isFullViewDialogOpen}
+          open={false}
```

Comment:
I think this Dialog can be removed.
There's also that onClickMore above that does nothing